.github/workflows/codeql-analysis.yml

```diff
@@ -52,6 +49,10 @@ jobs:
         uses: github/codeql-action/analyze@v3
         with:
           output: results
+      - name: Download add-on again
+        uses: actions/download-artifact@v4
+        with:
+          name: addon
```

Comment:
why does this download need to happen again?

Reply:
The download has to be performed again since the add-on has been moved to a zip file and then expanded, so the addon.nvda-addon file is not present..
I'll address this review ASAP, but perhaps it won't be finished today, in case you want this to be fixed quickly, feel free to do it yourself.

Reply:
Instead can you make a copy rather than rename the zip file?

Reply:
Done